components/base/kinematicbase/execution.go

```diff
@@ -96,8 +97,8 @@ func (ptgk *ptgBaseKinematics) GoToInputs(ctx context.Context, inputSteps ...[]r
 
 	// Pre-process all steps into a series of velocities
 	ptgk.inputLock.Lock()
-	ptgk.currentExecutingSteps, err = ptgk.arcStepsFromInputs(inputSteps, startPose)
-	arcSteps := ptgk.currentExecutingSteps
+	ptgk.currentState.currentExecutingSteps, err = ptgk.arcStepsFromInputs(inputSteps, startPose)
```

Comment:
Go style question.
It is preferred that when we do some `inputLock.Lock()` followed by some `inputLock.Unlock()` with some code in-between that we error check outside the input lock?
i.e. is it expected that the error of `ptgk.arcStepsFromInputs(inputSteps, startPose)` is written outside the lock?